eox_core/settings/common.py

```diff
@@ -46,3 +46,7 @@ def plugin_settings(settings):
             'fetch_from_created_on_site_prop',
             'fetch_from_user_signup_source',
         ]
+
+    # Sentry Integration
+    settings.EOX_CORE_SENTRY_INTEGRATION_ENABLED = False
+    settings.EOX_CORE_SENTRY_INTEGRATION_DSN = "fake_dsn"
```

Comment:
I think that instead of two separate configurations we can have one and use it at this way:

None if we want to disable that or "string" if we want to enable that.

Reply:
Agree, the change was applied.